packages/connect/src/types/settings.ts

```diff
@@ -22,7 +22,13 @@ export interface ConnectSettings {
     pendingTransportEvent?: boolean;
     lazyLoad?: boolean;
     interactionTimeout?: number;
-    // internal part, not to be accepted from .init()
+    coreMode?: 'auto' | 'popup' | 'iframe';
+    /* _extendWebextensionLifetime features makes the service worker in @trezor/connect-webextension stay alive longer */
+    _extendWebextensionLifetime?: boolean;
+}
+
+// internal part, not to be accepted from .init()
+export interface ConnectSettingsInternal {
```

Comment:
nice idea!

we do supply trustedHost from connect-explorer webextension setting page, on the other it is intended as internal. just a note.

Reply:
Ok so I'll move trustedHost to public to be safe